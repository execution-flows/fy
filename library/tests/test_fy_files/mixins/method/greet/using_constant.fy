method greeting using constant:
    def greet() -> None:
        print("Hello World!")
