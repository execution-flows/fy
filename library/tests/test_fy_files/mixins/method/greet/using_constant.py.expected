class Greet_UsingConstant_MethodMixin:
   def _greet(self) -> None:
        print("Hello World!")
